api/datastore/sql/sql.go

```diff
@@ -41,6 +48,7 @@ var tables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 	type varchar(16) NOT NULL,
 	headers text NOT NULL,
 	config text NOT NULL,
+	created_at text,
```

Comment:
why is the route the only subject of an audit?

Reply:
see other response, patch not about this